app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragment.kt

```diff
@@ -106,6 +109,28 @@ class HomeFragment : StdBrowseFragment(), AudioEventListener {
 			// Update the views before creating rows
 			views = callApi<ItemsResult> { apiClient.GetUserViews(TvApp.getApplication().currentUser!!.id, it) }
 
+			// Check for live TV support
+			if (TvApp.getApplication().currentUser!!.policy.enableLiveTvAccess) {
+				// This is kind of ugly, but it mirrors how web handles the live TV rows on the home screen
+				// If we can retrieve one live TV recommendation, then we should display the rows
+				callApi<ItemsResult> {
+					apiClient.GetRecommendedLiveTvProgramsAsync(
+						RecommendedProgramQuery().apply {
+							userId = TvApp.getApplication().currentUser!!.id
```

Comment:
Now that  3 places in this function use the currfentUser it might be worth adding it to a variable so we don't need to call the not-null operator each time.

`val currentUser = TvApp.getApplication().currentUser!!`